build.sbt

```diff
@@ -20,12 +20,12 @@ lazy val server = project
       Dependencies.scalaUri,
       Dependencies.squants,
       Dependencies.scalaguice,
+      Dependencies.commonsIO,
```

Comment:
The feature you need is from commons lang.  I think that's a separate dependency.

If it is please use commons labg here and keep commons IO as test dependency.

Relying on transitive dependencies is risky.

Reply:
Thanks for the catch! You are right about commons lang. I'm going to fix it in a separate PR.